### ISSUE TITLE
Viewer for this series settings not applying only to the series but for all

### ISSUE THREAD

**Issue by tmathura** [tachiyomiorg/tachiyomi#9416]
### Steps to reproduce

Prerequisite: the global Scale Type must be "Fit Screen".

1. Go to a manga series and open a chapter.
2. Click on the settings cog in the viewer that opens settings "For this series".
3. Change the Scale Type from "Fit Screen" to "Fit Width".
4. Go to a different manga series and open a chapter.
5. Open the settings "For this series".
6. The Scale Type is also "Fit Width" but it should be "Fit Screen"

### Expected behavior

When changing the Scale Type "For this series", it should not change the global Scale Type.

### Actual behavior

The Scale Type "For this series" changes the global Scale Type for all series.

### Tachiyomi version

0.14.6

### Android version

Android 13

### Device

Samsung Galaxy Z Fold3 5G

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have gone through the [FAQ](https://tachiyomi.org/help/faq/) and [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by arkon** [comment]
"For this series" only includes "Reading mode" and "Rotation type".

Per-series scaling has been declined back in #1849.
